### DISCUSSION TITLE
build  with agent node  fail

### DISCUSSION THREAD

**Discussion by mikeeastin** [Tencent/bk-ci#6424] [Q&A] [answered]
build log :
            Set up job
            Prepare checkout(checkout)
            Prepare POST：checkout(checkout)
            Start up the agent localhost/10.11.xx.xx [b-526b64c6ae4a4e3f86cf71fa45a2c27b]
            Terminate Plugin[Prepare_Job#1]: worker-agent.jar进程因未知原因意外退出，请检查构建机环境负载和agent目录是否完整
            Task [Prepare_Job#1] FAILED!
            [1]| Finish Job#1| worker-agent.jar进程因未知原因意外退出，请检查构建机环境负载和agent目录是否完整| status=FAILED

it seems  agent request   nginx  fail , the nginx log :
            [error] 31698#0: *15025 [lua] build_util.lua:71: auth_agent(): redis result is null, client: 10.1xx.xx, server: devops.devops, request: "PUT /ms/process/api/build/worker/started?retryCount=0 HTTP/1.1", subrequest: "/auth/build", host: "10.11.xx.xx"

**Comment by irwinsun** [comment, marked as answer]
鉴权失败， 看看redis配置与微服务配置是否一样？

/data/bkce/ci/gateway/core/lua/init.lua

/data/bkce/etc/ci/common.yml
